### PR TITLE
Update github.md

### PR DESCRIPTION
**Pull request by jerry-li-dev**
- Added section to call out static ip support.

- Need instructions for public users

### CODE REVIEW

File: src/pages/docs/integrations/available-integrations/github.md [hunk 1]
```diff
@@ -95,6 +95,10 @@ To finish, click **Add Integration**.
 
 Enter your GitHub Personal Access Token, specify your GitHub custom domain and click **Proceed**. In Backup your Postman Collections, choose an existing Postman collection, specifying your GitHub repository, a filename for your backup, and your chosen branch name.
 
+> If your network is behind a firewall that requires preapproved IP addresses, you will need to use a static IP address to enable backups to Github on custom domains.
```
Comment (shashankawasthi88) [inline src/pages/docs/integrations/available-integrations/github.md:98]: Can we reword it saying that "you will need to whitelist the static IP XXX in your firewall to enable the backup... " - "need to use does not give a very clear idea"

File: src/pages/docs/integrations/available-integrations/github.md [hunk 2]
```diff
@@ -95,6 +95,9 @@ To finish, click **Add Integration**.
 
 Enter your GitHub Personal Access Token, specify your GitHub custom domain and click **Proceed**. In Backup your Postman Collections, choose an existing Postman collection, specifying your GitHub repository, a filename for your backup, and your chosen branch name.
 
+> If your network is behind a firewall that requires whitelisted IP addresses, you will need to use a static IP address to enable backups to Github on custom domains.
```
Comment (shashankawasthi88) [inline src/pages/docs/integrations/available-integrations/github.md:98]: Same comments as the other file, All network calls needs to be corrected everywhere. Also "backup to Github" - it should be GitHub, I believe.

File: src/pages/docs/integrations/available-integrations/github.md [hunk 3]
```diff
@@ -95,6 +95,11 @@ To finish, click **Add Integration**.
 
 Enter your GitHub Personal Access Token, specify your GitHub custom domain and click **Proceed**. In Backup your Postman Collections, choose an existing Postman collection, specifying your GitHub repository, a filename for your backup, and your chosen branch name.
 
+>If your network is behind a firewall that requires whitelisted IP addresses, you will need to use a static IP address to enable backups to GitHub on custom domains.
+>Contact your IT team to whitelist the following static IP in your firewall to enable collection backup integrations:
+>US East: `3.212.102.200`
+>All calls for this integration will come from this IP address. Whitelisting this IP address will allow the integrations work as expected.
+
```
Comment (claireriley) [inline src/pages/docs/integrations/available-integrations/github.md:102]: Same recommendation as the GitLab one—I think we want to break this into a subsection so it's not all condensed in a note

**Review by claireriley — CHANGES_REQUESTED** [review]
Just formatting 😄

File: src/pages/docs/integrations/available-integrations/github.md [hunk 4]
```diff
@@ -95,10 +95,15 @@ To finish, click **Add Integration**.
...
 
->If your network is behind a firewall that requires whitelisted IP addresses, you will need to use a static IP address to enable backups to GitHub on custom domains.
->Contact your IT team to whitelist the following static IP in your firewall to enable collection backup integrations:
->US East: `3.212.102.200`
->All calls for this integration will come from this IP address. Whitelisting this IP address will allow the integrations work as expected.
+#### Static IP Support
+
+  If your network is behind a firewall that requires whitelisted IP addresses, you will need to use a static IP address to enable collection backups to GitHub on custom domains.
+
+  Contact your IT team to whitelist the following static IP in your firewall to enable collection backup integrations:
```
Comment (claireriley) [inline src/pages/docs/integrations/available-integrations/github.md]: Could we switch
> to enable collection backup integrations:

to

> to enable collection backups to GitHub

**Review by claireriley — APPROVED** [review]
One small suggestion! LGTM and feel free to merge when ready

**Pull request merged**